app/code/Magento/Backend/Helper/Dashboard/Order.php

```diff
@@ -18,62 +16,52 @@ class Order extends \Magento\Backend\Helper\Dashboard\AbstractDashboard
     /**
      * @var \Magento\Reports\Model\ResourceModel\Order\Collection
      */
-    protected $_orderCollection;
+    private $orderCollection;
 
     /**
      * @var \Magento\Store\Model\StoreManagerInterface
      * @since 100.0.6
      */
-    protected $_storeManager;
+    private $storeManager;
```

Comment:
This change is not backward compatible. Please revert it.
More details: https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development